src/openrct2/Context.cpp

```diff
@@ -574,6 +570,18 @@ namespace OpenRCT2
             OpenProgress(STR_LOADING_GENERIC);
         }
 
+        void InitialiseScriptEngine()
+        {
+#ifdef ENABLE_SCRIPTING
+            // TODO: add dedicated string
+            OpenProgress(STR_LOADING_GENERIC);
```

Comment:
What would be a good title for this? Just 'Initialising script engine...'?

Reply:
Maybe "Loading plugins..." would make more sense for non-developers?

Reply:
Hmm, the engine gets initialised even if you don't have any plugins installed, though. Perhaps 'Initialising plugin engine...' then?

Reply:
"Checking for plugins..." ? 😆